protocol/bin/flow.js

```diff
@@ -582,7 +646,7 @@ function lintJSON(json) {
 
 function lintError(s, lint) {
   if (lint === 'ignore') {
-    console.log('Ignoring lint error:', colors.yellow(s))
+    // console.log('Ignoring lint error:', colors.yellow(s))
```

Comment:
this is noisy and we don't even do anything in reaction to this so lets just silence